src/client.rs

```diff
@@ -34,6 +38,8 @@ impl Default for Options {
         Self {
             // Lean towards conformity
             crlf_relaxed: false,
+            // A value larger than the default `server::Options::max_command_size`.
```

Comment:
I'm not sure I understand this comment. Could you explain? Is the assumption just that responses are generally bigger? 100 MiB should okay, though.

Reply:
I just documented how I picked the number. I chose a value greater than the other value. Because I thought the client is usually less constrained than the server.

Reply:
I rephrased the comment.